backend/src/plugins/ModActions/ModActionsPlugin.ts

```diff
@@ -181,6 +182,12 @@ export const ModActionsPlugin = zeppelinGuildPlugin<ModActionsPluginType>()({
       };
     },
 
+    hasMutePermission(pluginData) {
+      return (member: GuildMember, channelId: string) => {
```

Comment:
This technically doesn't change the type, but why not 

```suggestion
      return (member: GuildMember, channelId: Snowflake) => {
```